pkg/codegen/internal/test/testdata/resource-property-overlap/schema.json

```diff
@@ -12,9 +12,9 @@
   },
   "language": {
     "csharp": {
-      "packageReferences": {
-        "Pulumi": "3.12"
-      }
+      "projectReferences": [
```

Comment:
What's the reasoning behind this change? Why do some schemas use a package reference while others use a project reference?

Reply:
Ideally, all schemas should use a `projectReference` and refer to the sdk that they are compiling against. That way we can catch when we break codegen by changing the schema. I don't remember why I changed this specific schema. 

The reason we had packageReferences originally AFAIK, is because we didn't support the `projectReference` option in the schema.